libs/runtime/ebpf_epoch.c

```diff
@@ -384,7 +394,7 @@ ebpf_epoch_exit(_In_ ebpf_epoch_state_t* epoch_state)
         KIRQL saved_irql = epoch_state->irql_at_enter;
         epoch_state->irql_at_enter = DISPATCH_LEVEL;
 
-        _ebpf_epoch_send_message_and_wait(&message, epoch_state->cpu_id);
+        _ebpf_epoch_send_message_and_wait(&message, epoch_state->cpu_id, true);
```

Comment:
(minor) to make this more readable either use a constant/flag with meaningful name or qualify the true literal with parameter name in comment.